### PR TITLE
LPS-97223 Update lockfile

### PR DESCRIPTION
**Pull request by diegonvs**
During this upgrade, I identified a bug that is caused by [bootstrap.js](https://github.com/twbs/bootstrap/blob/v4.3.1/js/src/dropdown.js#L484) that is treating accessibility triggering a click event when using the keyboard but in this case, we needed to avoid these treatments and use only the implementation guided by the Clay component.

We tried to overwrite this [function](https://github.com/twbs/bootstrap/blob/v4.3.1/js/src/dropdown.js#L484) behavior, setting up a hack on [`liferay.js`](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/liferay.js#L67) file but we failed when doing this.

### CODE REVIEW

**Comment by diegonvs** [comment]
ci:test:sf

**Comment by diegonvs** [comment]
ci:test:relevant

**Comment by diegonvs** [comment]
ci:stop

**Comment by diegonvs** [comment]
ci:test:sf

**Comment by diegonvs** [comment]
ci:test:relevant

**Comment by diegonvs** [comment]
ci:test:relevant

**Comment by brunobasto** [comment]
if we could find another way to make bootstrap.js not mess with Clay's autocomplete, that would be better. I'm fine with adding this hack to the Text field, but that issue will happen whenever Clay Autocomplete is used inside portal (right now I guess we are the only ones using it and hopefully not for long)

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Text/Text.es.js [hunk 1]
```diff
@@ -16,11 +16,17 @@ import '../FieldBase/FieldBase.es';
...
 import Component from 'metal-component';
+import dom from 'metal-dom';
 import Soy from 'metal-soy';
 import templates from './Text.soy.js';
 import {Config} from 'metal-state';
 
 class Text extends Component {
+	attached() {
+		const portalElement = dom.toElement('#clay_dropdown_portal');
+		dom.addClasses(portalElement, 'show');
```
Comment (brunobasto) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Text/Text.es.js]: Not sure if that could happen, but some defensive coding could prevent issues in the future. What happens if, for some reason, `#clay_dropdown_portal` is not on the page?

Comment (diegonvs) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Text/Text.es.js]: A conditional for checking if the portal element exists was added.

**Comment by diegonvs** [comment]
I have one doubt: I applied this fix for the components where this component is being used, I don't think it's right to add this workaround for Clay 2.x due to the specificity of this issue: We're trying to use keyboard keys on ClayAutocomplete when `bootstrap.js` is loaded on the page. What are your thoughts? /ccing @jbalsas @matuzalemsteles @brunobasto

**Comment by diegonvs** [comment]
Hey @ambrinchaudhary, I've updated your module, could you take a look if everything is okay?

**Comment by diegonvs** [comment]
ci:test:relevant

**Comment by diegonvs** [comment]
ci:test:sf

**Comment by ambrinchaudhary** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/sharing/sharing-web/src/main/resources/META-INF/resources/Sharing.es.js [hunk 2]
```diff
@@ -287,6 +288,13 @@ class Sharing extends PortletBase {
 
 		return valid;
 	}
+
+	attached() {
```
Comment (ambrinchaudhary) [inline modules/apps/sharing/sharing-web/src/main/resources/META-INF/resources/Sharing.es.js]: Hey @diegonvs , shouldn't be this at the beginning of the class?

**Comment by ambrinchaudhary** [comment]
> Hey @ambrinchaudhary, I've updated your module, could you take a look if everything is okay?

Hey @diegonvs, I have tested our autocomplete component in Sharing and now works as expected! 👍 . 
I just let you a comment above. 
Thank you!

File: modules/apps/sharing/sharing-web/src/main/resources/META-INF/resources/Sharing.es.js (hunk 2 above)
Comment (diegonvs) [reply modules/apps/sharing/sharing-web/src/main/resources/META-INF/resources/Sharing.es.js]: Yes! I missed!

**Comment by diegonvs** [comment]
ci:test:sf

**Comment by diegonvs** [comment]
ci:test:relevant

**Comment by diegonvs** [comment]
ci:test:relevant

**Comment by jbalsas** [comment]
Pull request submitted to https://github.com/jbalsas/liferay-portal/pull/1820. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**